photospp-toolfile.spec

```diff
@@ -1,4 +1,4 @@
-### RPM external photospp-toolfile 3.56
+### RPM external photospp-toolfile 2.0
```

Comment:
right version in toolfile has nothing to do with version of tool. toolfile version should only be changed when the tool definition is changed